runner/runner.go

```diff
@@ -49,29 +49,22 @@ func (c Command) String() string {
 	return b.String()
 }
 
-func NewCommand(argv []string, env map[string]string, timeout time.Duration, snapshotId string) *Command {
-	return &Command{Argv: argv, EnvVars: env, Timeout: timeout, SnapshotId: snapshotId}
+func NewCommand(argv []string, env map[string]string, timeout time.Duration) *Command {
```

Comment:
Oh, are you saying we should change Command to RunRequest?

I think it turns out that aybe we don't need this function at all, cause you can just create a Command.